### ISSUE TITLE
Gateway Log issues even with updated Dis.c.um

### ISSUE THREAD

**Issue by pregernet** [vivinano/MudaeAutoBot#101]
I've made absolutely sure that Dis.c.um is updated to its most recent version, and I still receive the following gateway issues when executing MudaeAutoBot.py. I'm sorry to post about non-bot related issues but I cannot seem to find a workaround.

![image](https://user-images.githubusercontent.com/91040006/133953583-f9bc3caf-36e9-4979-ba48-6ff96b2cf03d.png)

**Comment by vivinano** [comment]
It seems like you haven’t used a user token so it’s not able to log in so it’s returning a blank user 

If you want post a partial or like a blurred picture of your user token so I can see if it’s correct

**Comment by pregernet** [comment]
> It seems like you haven’t used a user token so it’s not able to log in so it’s returning a blank user If you want post a partial or like a blurred picture of your user token so I can see if it’s correct
> […](#)
> Sent from my iPhone
> On Sep 19, 2021, at 11:10 PM, pregernet ***@***.***> wrote: ﻿ I've made absolutely sure that Dis.c.um is updated to its most recent version, and I still receive the following gateway issues when executing MudaeAutoBot.py. I'm sorry to post about non-bot related issues but I cannot seem to find a workaround. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

![image](https://user-images.githubusercontent.com/91040006/133961616-391661e2-27d0-4356-8597-c5d7a340ba98.png)

here it is! thanks so much

**Comment by vivinano** [comment]
Seems like a token is it a user token or a bot token ? Because the discum library only functions with user tokens

**Comment by vivinano** [comment]
just checking in on this issue ?

**Comment by vivinano** [comment]
I havent heard response back for 2 days will be closing issue

TLDR at this point is assuming bot token was used and not a user token
